tests/test_logger.py

```diff
@@ -250,6 +251,62 @@ def test_report_video_to_unsupported_format_raises_error(tmp_path, unsupported_f
     writer.close()
 
 
+_called = None
+
+
+def get_fail_first_then_pass_fn(fn, exception=Exception):
+    _called = False
+
+    @functools.wraps(fn)
+    def _fn(*args, **kwargs):
+        global _called
+        if not _called:
+            _called = True
+            raise exception()
+        return fn(*args, **kwargs)
+
+    return _fn
+
+
+@pytest.mark.parametrize(
+    "histogram,fail_first_write",
+    [
+        (th.rand(100), False),
+        (np.random.rand(100), False),
+        (np.random.rand(100), True),
+    ],
+)
+def test_report_histogram_to_tensorboard(tmp_path, read_log, fail_first_write, histogram):
+    pytest.importorskip("tensorboard")
+
+    writer = make_output_format("tensorboard", tmp_path)
+
+    if fail_first_write:
+        writer.writer.add_histogram = get_fail_first_then_pass_fn(writer.writer.add_histogram, TypeError)
+
+    writer.write({"data": histogram}, key_excluded={"data": ()})
+
+    log = read_log("tensorboard")
+
+    assert not log.empty
+    assert any("data" in f for f in log.lines)
+    assert any("Histogram" in f for f in log.lines)
+
+    writer.close()
+
+
+@pytest.mark.parametrize("histogram", [list(np.random.rand(100)), tuple(np.random.rand(100)), "1 2 3 4"])
+def test_report_unsupported_type_as_histogram_to_tensorboard(tmp_path, read_log, histogram):
```

Comment:
hmm, this just checks that nothing is logged, it isn't really specific to histogram, right?